bukuserver/templates/bukuserver/statistic.html

```diff
@@ -333,8 +363,6 @@ <h4 class="modal-title" id="myModalLabel">Title ranking</h4>
       }
     }
   });
-
-  titleChart.canvas.parentNode.style.height = '128px';
```

Comment:
This wasn't doing anything other than causing rendering issues in some cases :sweat_smile: